### ISSUE TITLE
About how to increase the number of the new classes？

### ISSUE THREAD

**Issue by iRuizeWu** [MasLiang/Learning-without-Forgetting-using-Pytorch#7]
## **First question：**
I did not understand how you keep adding new classes to your code.
"# only use the first class" is mentioned in line 31 of your "trainloader" source code. what does that mean?
Does it mean: If I want to add a new class or multiple new classes, I need to manually change the "1" to "2, 3" and so on after training 200 epochs.
## Second question:
In "main.py", line 151 "targets += out_features". What's the meaning? Why targets should be added to the output number of the last layer of the old network. Your "targets" should be a tensor of minibatch labels. Add them up, I didn't understand.
## The third question:
In the original "LOSS", there are three parts of the loss value, so I want to discuss with you how you view the "R" regularization term.

**Comment by iRuizeWu** [comment]
Q2: i know it.

**Comment by MagicHealer** [comment]
@iRuizeWu Can you explain this to me? Have you successfully replicated this? I cannot replicate it properly. Thank you
